src/css/finalPropName.js

```diff
@@ -17,17 +19,6 @@ function vendorPropName( name ) {
 			return name;
 		}
 	}
-}
 
-// Return a potentially-mapped vendor prefixed property
-export function finalPropName( name ) {
-	var final = vendorProps[ name ];
-
-	if ( final ) {
-		return final;
-	}
-	if ( name in emptyStyle ) {
-		return name;
-	}
-	return vendorProps[ name ] = vendorPropName( name ) || name;
+	return name;
```

Comment:
Just a reminder: as stated in https://github.com/jquery/jquery/pull/5583#issuecomment-2487013528, we have to avoid overwriting the `name` parameter, line 17 above.